mgo.go

```diff
@@ -422,7 +423,16 @@ func MgoTestPackage(t *testing.T, certs *Certs) {
 	gc.TestingT(t)
 }
 
+// Output implements the mgo log_Logger interface.
+func (s *MgoSuite) Output(calldepth int, message string) error {
```

Comment:
I don't mind too much, but this doesn't really belong on the suite. A simpler wrapper type would be cleaner. The logger wouldn't even have to be added as a field on MgoSuite...